### ISSUE TITLE
Bug: Level: "Unblocked"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#107]
An Android only level (bug).

```
:name=Unblocked
:description=Get out of my way (Android only)
:num_rabbits=2
:num_to_save=1
:music=tryad-witness
   Q  k      
#         O #
#############
```

**Comment by tttppp** [comment]
To be clear - the initial aim is to work out how to solve the level. After that the aim is to fix the bug (but that's probably less fun).

**Comment by tttppp** [comment]
I'm really sorry, the number to save should be 1 on this level. I've also not tried this exact level, but I've seen the behaviour on my mobile.

**Comment by andybalaam** [comment]
Still haven't managed to try it but still looking forward to doing so :-)

**Comment by tttppp** [comment]
I fixed the number to save in the initial level file.

**Comment by tttppp** [comment]
I expect this issue is now fixed (see #403). In case you didn't work it out, the solution to this was to rotate your Android device at the correct time. Occasionally this reorder the rabbits somehow, or causes the blocking check to be skipped, and the rabbit passes straight through the blocker.

**Comment by andybalaam** [comment]
Nice. I'll have to try and reproduce it

**Comment by andybalaam** [comment]
(Before and after the fix)

**Comment by andybalaam** [comment]
Assuming this is fixed due to #403 - re-open if not.
